### DISCUSSION TITLE
There was an issue joining the organization: Your GitHub user account @UserName is currently unlinked. However, you are attempting to authenticate with your Identity Provider using the 'UserEmail' SAML identity which is already linked to a different GitHub user account in the organization. Please reach out to one of your GitHub organization owners for assistance.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#55825] [Accessibility] [answered]
### Select Topic Area

Question

### Body

Hi,
After 2-factor login via okta to GitHub, I am getting below error. Could you please confirm me why I am getting this error.

There was an issue joining the organization: Your GitHub user account @UserName is currently unlinked. However, you are attempting to authenticate with your Identity Provider using the 'UserEmail' SAML identity which is already linked to a different GitHub user account in the organization. Please reach out to one of your GitHub organization owners for assistance.

**Comment by mkhavari01** [comment, marked as answer]
It seems that you are encountering an error while trying to join an organization on GitHub. The error message suggests that your GitHub account is currently unlinked, but the SAML identity you are trying to authenticate with is already linked to a different GitHub account in the organization.

Here are some possible causes for this error:

You might have multiple GitHub accounts with different email addresses, and the SAML identity you are using belongs to a different account than the one you are trying to join the organization with.

You might have previously linked your SAML identity to a different GitHub account, and the organization is trying to link it to a different account.

There might be a configuration issue with the SAML identity provider or the GitHub organization that is causing the error.

To resolve this issue, you should reach out to one of your GitHub organization owners for assistance. They should be able to help you diagnose the issue and provide guidance on how to resolve it.

**Reply by mkhavari01** [reply]
please approve the answer if it was helpful

**Reply by ghost** [reply]
Thanks!

**Reply by arunjohney-freightways** [reply]
_You might have previously linked your SAML identity to a different GitHub account, and the organization is trying to link it to a different account._

As the organization owner how can one resolve the issue caused by the above scenario?

**Reply by aochsner** [reply]
https://github.com/orgs/community/discussions/79261#discussioncomment-8020950

**Reply by davidthomas2063** [reply]
As an Enterprise owner, I cant find any accounts in any of my orgs which has the SAML email linked to it. I did an export of all users to be sure. If I cant find the an account with this email connected, should I contact GitHub support?
